src/pages/workspace/companyCards/assignCard/TransactionStartDateStep.tsx

```diff
@@ -143,13 +117,22 @@ function TransactionStartDateStep({policyID, feed, backTo}: TransactionStartDate
                         />
                     }
                     listFooterContent={
-                        <TransactionStartDateSelectionListFooter
-                            dateOptionSelected={dateOptionSelected}
-                            startDate={startDate}
-                            policyID={policyID}
-                            feed={feed}
-                            backTo={backTo}
-                        />
+                        dateOptionSelected === CONST.COMPANY_CARD.TRANSACTION_START_DATE_OPTIONS.CUSTOM ? (
+                            <View style={[styles.ph5]}>
+                                <DatePicker
+                                    inputID=""
+                                    value={startDate}
+                                    label={translate('iou.startDate')}
+                                    onInputChange={(value) => {
+                                        setErrorText('');
+                                        setStartDate(value);
+                                    }}
```

Comment:
In https://github.com/Expensify/App/issues/76108, we decided that we should also add validation on input change.